packages/visual-editor/src/vite-plugin/defaultLayoutData.ts

```diff
@@ -1781,7 +1781,7 @@ const mainDefaultLayout = {
                                   image: {
                                     field: "",
                                     constantValue: {
-                                      url: "https://images.unsplash.com/photo-1755745360285-0633c972b0fd?ixlib=rb-4.1.0&q=85&fm=jpg&crop=entropy&cs=srgb&w=1000",
+                                      url: "https://placehold.co/80x80",
```

Comment:
I think we want the unsplash image in both cases: default layout and dragging component